webviz_subsurface/containers/_inplace_volumes_onebyone.py

```diff
@@ -1,5 +1,6 @@
 from uuid import uuid4
 import json
+from pathlib import Path
```

Comment:
While we are here, put `import json` also before `from uuid import uuid4` as that is shorter and not using `from`?